### PR TITLE
Gerben apply to v1.5

### PR DESCRIPTION
**Pull request by gerben-stavenga**
### Description

Apply patch to v1.5 public repo for historical consistency

### CODE REVIEW

File: types/src/on_chain_config/timed_features.rs [hunk 1]
```diff
@@ -58,6 +60,8 @@ impl TimedFeatureFlag {
         match (self, chain_id) {
             (DisableInvariantViolationCheckInSwapLoc, TESTNET) => NOT_YET_SPECIFIED,
             (DisableInvariantViolationCheckInSwapLoc, MAINNET) => NOT_YET_SPECIFIED,
+            (LimitTypeTagSize, TESTNET) => 1689966000000,
```
Comment (perryjrandall) [inline types/src/on_chain_config/timed_features.rs:63]: I think you need to remove these

Comment (gerben-stavenga) [reply types/src/on_chain_config/timed_features.rs:63]: This is historic right? So this is to the best of the our knowledge what was final release of v1.5

**Review by perryjrandall — APPROVED** [review]
accepting to unblock

File: types/src/on_chain_config/timed_features.rs (hunk 1 above)
Comment (perryjrandall) [reply types/src/on_chain_config/timed_features.rs:63]: In that case maybe we can include a human readable timestamp

**Review by davidiw — APPROVED** [review]

**Pull request merged**